Library/Homebrew/os/mac/xcode.rb

```diff
@@ -15,7 +15,7 @@ def latest_version
         when "10.8"  then "5.1.1"
         when "10.9"  then "6.2"
         when "10.10" then "7.2.1"
-        when "10.11" then "7.3.1"
+        when "10.11" then "8.0"
```

Comment:
It's not the only way to have the CLT. Apple have noted in Xcode 8.1's release notes that if you want the CLT on 10.11 you should be using the 7.3.1 CLT, not any 8.x CLT (which doesn't exist for 10.11). Presumably from that they're perfectly happy for people to mix & match Xcode/CLT versions going forwards.

Also worth noting, if you only have the CLT and not Xcode, you won't be forced to update anything.